HELM-307: Grafana Location Dashboard - Performance (#389)
OpenNMS/opennms-helm: docs/modules/datasources/pages/performance_datasource.adoc

You can then select from the list to change the data the panel displays.
See xref:panel_configuration:dynamic-dashboard.adoc#pc-template-filters[Using Template Variables to Create Dynamic Dashboards] for an example.

> === nodeLocation()

Reviewer: IMO, this should be plural. It does not give the location of a node but all relevant location**s**.

Reviewer: Thinking some more, this should be `locations` only. By wording, there is no "Node Location h but only a "Location" where Nodes (and other stuff) are located in.

Author: Applied change

> The `nodeLocation()` query will return a list of all available nodes locations. 

Reviewer 2: ```suggestion
The `nodeLocation()` query returns a list of all available node locations. 
```

> This variable is meant to be used together with nodeFilter, in order to reduce the number of nodes displayed by location.

Reviewer 2: ```suggestion
Use this variable with nodeFilter to reduce the number of nodes displayed by location.
```
